CHANGELOG.latest.md

```diff
@@ -1,29 +1,44 @@
 ## RevenueCat SDK
-### ✨ New Features
-* Paywalls v2 sheet anysize (#5056) via Josh Holtz (@joshdholtz)
-* Purchase web package (#5049) via Josh Holtz (@joshdholtz)
+### 🐞 Bugfixes
+* Fix offerings not being returned in the `offerings` property of the SDK Health Report (#5043) via Pol Piella Abadia (@polpielladev)
+### Customer Center
+#### ✨ New Features
+* Add management URL to PurchaseInformation (#5080) via Facundo Menzella (@facumenzella)
+* feat: Show subscription list instead of only the active subscription (#5050) via Facundo Menzella (@facumenzella)
```

Comment:
This is actually not part of **5.24.0**. I've reverted it @joshdholtz .

Can we just edit this file?